### PR TITLE
Chana docs 3

### PR DESCRIPTION
**Pull request by chanagershom**
This PR includes the addition of a User Management page under Cloudify Manager, addition of a Creating Widgets page under the UI and updates to the Multi-tenancy page under Cloudify Manager. Please verify all three pages.

### CODE REVIEW

File: content/manager/user-management.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+### How Cloudify Manager Works with the LDAP/AD Service
+
+When a user logs in to Cloudify Manager, their credentials are passed to the LDAP/AD service for authentication. By default, all users in the LDAP/AD service are authenticated to Cloudify Manager, however only users who have specific permissions for a tenant can access it.
+
+When a user logs into Cloudify Manager, the service authenticates the user and returns a list of any groups to which the user belongs.
+
+In Cloudify Manager, if you have added a group to a tenant, using the process that complies with the requirements for defining a group specified in LDAP/AD, all users in that LDAP group can access the tenant (unless an individual user is specifically suspended from the tenant.) The For more information about specifying LDAP/AD-compliant user-group names, see Adding Users on the Tenant Management page.
```
Comment (talisela) [inline content/manager/user-management.md]: We don't currently have the option of suspending a specific user from a tenant, we can remove this comment.

File: content/manager/user-management.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+For more information, see the [users]({{< relref "cli/users.md" >}}) and [user-groups]({{< relref "cli/usergroups.md" >}}) commands in the CLI documentation.
+
+
+#### Tenant-Related Commands
+
+You can add and remove users and user groups to/from a specific tenant. To run these user-related tenant-specific commands, use `cfy tenant`. For more information, see the [tenants]({{< relref "cli/tenants.md" >}}) command in the CLI documentation.
+
+- `add-user` enables you to add an individual user to a tenant
+- `add-user-group` enables you to add a group of users to a tenant
```
Comment (talisela) [inline content/manager/user-management.md]: That's a bit confusing - it allows you to add user-groups to the tenant, not a group of users.

File: content/manager/user-management.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- `create` enables you to create a tenant
+- `delete` enables you to delete a tenant
+- `get` enables you to view information about a tenant, including its users
+- `remove-user` enables you to remove a specific user from a group on the tenant
```
Comment (talisela) [inline content/manager/user-management.md]: should be:
enables you to remove a specific user from a **tenant**

File: content/manager/user-management.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+#### Tenant-Related Commands
+
+You can add and remove users and user groups to/from a specific tenant. To run these user-related tenant-specific commands, use `cfy tenant`. For more information, see the [tenants]({{< relref "cli/tenants.md" >}}) command in the CLI documentation.
+
+- `add-user` enables you to add an individual user to a tenant
+- `add-user-group` enables you to add a group of users to a tenant
+- `create` enables you to create a tenant
+- `delete` enables you to delete a tenant
+- `get` enables you to view information about a tenant, including its users
```
Comment (talisela) [inline content/manager/user-management.md]: Why did we remove the "list" explanation?
I think it should be - 

provides a list of all tenants in this instance of Cloudify Manager  
 -  By default, when you generate the list of tenants, only the number of linked user-groups and users is displayed. You can retrieve full details with the use of a `--get-data` flag.

**Comment by chanagershom** [comment]
Yes, we can wait but the other bits are also delayed so let's resolve ASAP.

Regards,
Chana Gershom
Technical Writer
Ph. 054 7718415

**Pull request merged**